packages/@react-spectrum/overlays/src/Overlay.tsx

```diff
@@ -31,6 +31,10 @@ function Overlay(props: OverlayProps, ref: DOMRef<HTMLDivElement>) {
     onExited,
     nodeRef
   } = props;
+
+  let {getContainer} = useUNSTABLE_PortalContext();
+  container = container || getContainer?.();
```

Comment:
I think ideally this would go in the React Aria Overlay component rather than the RSP one? I see the context in `@react-aria/overlays` but nothing in there uses it.